src/test/java/org/prebid/server/bidder/adtrgtme/AdtrgtmeBidderTest.java

```diff
@@ -0,0 +1,287 @@
+                    .containsExactly(
+                            tuple(HttpUtil.CONTENT_TYPE_HEADER.toString(), HttpUtil.APPLICATION_JSON_CONTENT_TYPE),
+                            tuple(HttpUtil.ACCEPT_HEADER.toString(), HttpHeaderValues.APPLICATION_JSON.toString()),
+                            tuple(HttpUtil.X_OPENRTB_VERSION_HEADER.toString(), "2.5"));
+            assertThat(httpRequest.getPayload()).isEqualTo(bidRequest);
+        });
+    }
+
+    @Test
+    public void makeHttpRequestsShouldCreateExpectedUrlWithSiteId() {
+        String siteId = "site_id";
+        // given
+        final BidRequest bidRequest = BidRequest.builder().id("test-request-id").imp(singletonList(
+                givenImp(identity()))).site(Site.builder().id(siteId).build()).build();
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = adtrgtmeBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(1)
+                .extracting(HttpRequest::getUri)
+                .containsExactly("https://z.cdn.adtarget.market/ssp?s=site_id&prebid");
+    }
+
+    @Test
+    public void makeHttpRequestsShouldCreateExpectedUrlWithAppId() {
+        String appId = "app_id";
+        // given
+        final BidRequest bidRequest = BidRequest.builder().id("test-request-id").imp(singletonList(
+                givenImp(identity()))).app(App.builder().id(appId).build()).build();
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = adtrgtmeBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isEmpty();
+        assertThat(result.getValue()).hasSize(1)
+                .extracting(HttpRequest::getUri)
+                .containsExactly("https://z.cdn.adtarget.market/ssp?s=app_id&prebid");
+    }
+
+    @Test
+    public void makeHttpRequestsShouldReturnErrorSiteAppIdsNotProvided() {
+        // given
+        final BidRequest bidRequest = BidRequest.builder().id("test-request-id").imp(singletonList(
+                givenImp(identity()))).build();
+
+        // when
+        final Result<List<HttpRequest<BidRequest>>> result = adtrgtmeBidder.makeHttpRequests(bidRequest);
+
+        // then
+        assertThat(result.getErrors()).isNotEmpty();
+        assertThat(result.getErrors().get(0).getMessage()).startsWith("request.Site or request.App are not provided");
+    }
+
+    @Test
+    public void makeHttpRequestShouldReturnTwoHttpRequestsWhenTwoImpsHasDifferentSourceId() {
+        // given
+        final BidRequest bidRequest = BidRequest.builder().site(Site.builder().id("site_id").build())
+                .imp(asList(Imp.builder()
+                                .xNative(Native.builder().build())
+                                .ext(mapper.valueToTree(
+                                        ExtPrebid.of(null, ExtImpAdrino.of("test"))))
+                                .build(),
```

Comment:
I can see that you create imps by hand many times in this test, maybe it will be cleaner if you create helper method like
`givenImp(impCustomizer)`?